.circleci/config.yml

```diff
@@ -1112,16 +1112,16 @@ workflows:
           requires:
             - build
             - mock_e2e_tests
-      - integration_test:
-          filters:
-            branches:
-              only:
-                - master
-                - graphqlschemae2e
-                - beta
-          requires:
-            - build
-            - mock_e2e_tests
+      # - integration_test:
+      #     filters:
+      #       branches:
+      #         only:
+      #           - master
```

Comment:
nit I think this is the only line you need to comment

Reply:
both master and beta right?